### PR TITLE
Add sidekiq to content tagger

### PR DESCRIPTION
**Pull request by mobaig**

### CODE REVIEW

File: hieradata/common.yaml [hunk 1]
```diff
@@ -259,6 +259,9 @@ govuk::apps::content_store::nagios_memory_critical: 1300
 govuk::apps::content_tagger::db_hostname: "postgresql-primary-1.backend"
 govuk::apps::content_tagger::db::backend_ip_range: "%{hiera('environment_ip_prefix')}.3.0/24"
 
+govuk::apps::content_tagger::redis_host: 'redis-1.backend'
```
Comment (deanwilson) [inline hieradata/common.yaml]: You don't need these two lines as the defaults in the puppet class will set them correctly.

**Comment by mobaig** [comment]
@deanwilson @tijmenb just updated this - how's it looking?

**Comment by tijmenb** [comment]
👍 merge away

**Comment by andrewgarner** [comment]
FYI I've got a [PR](https://github.com/alphagov/sidekiq-monitoring/pull/25) in the pipeline that should make it easier for you to monitor this.

**Pull request merged**